examples/mini-hlint/src/Main.hs

```diff
@@ -259,7 +267,9 @@ main = do
 #endif
         ]
     harvestAnns pst =
```

Comment:
Can we just not define this instead of setting it to `undefined`? Afaict, the callsite is guarded behind `!defined GHC_MASTER` so I don’t see why we need this there.

Reply:
Thanks, yes that's better. Just building here and then I'll push an amendment.